pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -544,3 +547,32 @@ func (exp *flowExporter) sendDataSet() (int, error) {
 	klog.V(4).Infof("Data set sent successfully. Bytes sent: %d", sentBytes)
 	return sentBytes, nil
 }
+
+func (exp *flowExporter) findFlowType(record flowexporter.FlowRecord) uint8 {
+	// TODO: support Pod-To-External flows in network policy only mode.
+	if exp.isNetworkPolicyOnly {
+		if record.Conn.SourcePodName == "" || record.Conn.DestinationPodName == "" {
+			return ipfixregistry.InterNode
```

Comment:
you probably should rename these to `FlowTypeInterNode`, etc. in go-ipfix so there is more context in the name

Reply:
As these were used in the context of adding flow types (the function is GetFlowType here), I thought a simple name might be sufficient. If it's confusing, we should definitely change it. 
@dreamtalen Could you please create an issue in go-ipfix?

Reply:
Sure, thanks.